src/components/Button.js

```diff
@@ -135,30 +136,47 @@ const Button = React.createClass({
         ':hover': {
           color: StyleConstants.adjustColor(this.props.primaryColor, -8),
           fill: StyleConstants.adjustColor(this.props.primaryColor, -8),
-          transition: 'all .2s ease-in'
+          transition: 'all .2s ease-in',
+          borderColor: StyleConstants.Colors.FOG,
+          borderRadius: 2,
+          borderWidth: 1
```

Comment:
One gotcha on using shorthand styles is that if it is something that you might be overwriting, you can run into issues. Radium throw a warning if you mix long and short hand. To if you use shorthand for the border, and then pass in a prop to change just the width, it complains.